### PR TITLE
Encryptor + encrypted_secret

### PR DESCRIPTION
**Pull request by ngw**
Encrypts secrets.

ENV variables must be added to the .env file: `SECRET_KEY_BASE` and `ENCRYPTOR_SALT`.
I've generated my own key_base with `rake secret` and the salt with https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/P4-951_storing-secrets/spec/services/encryptor_spec.rb#L9

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -6,6 +6,9 @@ NOMIS_AUTH_SCHEME='auth_scheme'
 NOMIS_API_PATH_PREFIX='/api_prefix'
 NOMIS_AUTH_PATH_PREFIX='/auth_prefix'
 
+SECRET_KEY_BASE="bccfda6a964030d97c6bd37447ad6aeedb6d03c2fab2f4a1b54713e60aa84d621a13105ace4413a7b505e6e6e8269b78518ba426585992e664c7ec1f711f61a5"
+ENCRYPTOR_SALT="\x97\x19\xBF\xF6\xC2\x9DZ\x92D\xFFq\xFC\x9B\x1C\xC4\t\x95\xBA\xBF\xF7q<\xC9\xFC\x81\xBD;\xDEtH\xA4\xB2
```
Comment (martyn-w) [inline .env.example]: seem to be missing a `"`

**Comment by martyn-w** [comment]
👍  even better

**Comment by spikeheap** [comment]
@ngw did you mean to add swagger/v1/swagger.yaml here? It looks like it's been removed in a recent commit (but maybe not added to .gitignore)

**Comment by martyn-w** [comment]
@spikeheap good spot, I've removed it and added it to `.gitignore` in branch https://github.com/ministryofjustice/hmpps-book-secure-move-api/tree/P4-950-webhook-jobs - which this will shortly be merged with

**Pull request merged**